docs/tutorial/visualization/visualizer_advanced.rst

```diff
@@ -0,0 +1,359 @@
+    0
+
+And what is its mesh name?
+
+.. code-block:: python
+
+    >>> helmet.meshes[0].mesh_name
+    'Hose_low'
+
+
+We can write a loop which displays all mesh names and material indices used in a complex model like so:
+
+
+.. code-block:: python
+
+    >>> for m in helmet.meshes:
+    ...     print(m.mesh_name)
+    ...     print(m.material_idx)
+    ... 
+    Hose_low
+    0
+    RubberWood_low
+    1
+    GlassPlastic_low
+    2
+    MetalParts_low
+    3
+    LeatherParts_low
+    4
+    Lenses_low
+    5
+  
+   
+
+We can also render meshes individually like:
+
+.. code-block:: python
+
+    >>> vis.draw(helmet.meshes[0].mesh)
+    
+.. image:: https://user-images.githubusercontent.com/93158890/149238095-5385d761-3bae-4172-ab45-1d47b6084d5c.jpg
+    :width: 700px
+
+
+Rendering sub-models
+::::::::::::::::::::
+
+
+Just like in the previous loop example which displays all ``mesh_name`` and ``material_idx`` properties, we can write a loop which renders each mesh separately:
+
+.. code-block:: python
+
+    >>> for m in helmet.meshes:
+    ...     vis.draw(m.mesh)
+    
+A series of Open3D visualizer windows should appear. As you close each of them, a new one will appear with a different mesh:
+
+1) A hose:
+
+.. image:: https://user-images.githubusercontent.com/93158890/149238208-961a0a8d-ebb2-4621-aff1-8bfcdeced734.jpg
+    :width: 700px
+    
+2) All wooden and rubber parts:
+
+.. image:: https://user-images.githubusercontent.com/93158890/149238298-98a894cd-72a2-4c76-8e30-da89e26f2fa4.jpg
+    :width: 700px
+
+Other parts will follow:
+
+3) The goggles and earphones parts
+4) All metallic parts
+5) Leather parts
+6) Lenses
+
+Cool, isn't it? Now, we can modify the same loop to display all materials and associated properties:
+
+.. code-block:: python
+
+    >>> for m in helmet.meshes:
+    ...     vis.draw({'name' : m.mesh_name, 'geometry' : m.mesh, 'material' : helmet.materials[m.material_idx]})
+
+This will give us a full display of each part:
+
+1) A hose:
+
+.. image:: https://user-images.githubusercontent.com/93158890/149238906-065fad20-ed3f-4585-b90b-7d30b5c06912.jpg
+    :width: 700px
+    
+2) All wooden and rubber parts (breathing mask):
+
+.. image:: https://user-images.githubusercontent.com/93158890/149239024-e361bb4a-5fe5-44e7-b41d-8b6d777a1b9b.jpg
+    :width: 700px
+
+And other parts, just like in the previous ``helmet.meshes`` loop:
+
+3) The goggles and earphones parts    
+4) All metallic parts
+5) Leather parts:
+6) Lenses
+
+
+
+Rendering a ``Tensor``-based ``TriangleMesh`` monkey
+::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+In the beginning of this tutorial (:ref:`rendering_models`), we rendered a ``TriangleMesh`` of a monkey model using the ``o3d.io.read_triangle_mesh()`` method. Now, we will modify our earlier exercise to convert regular ``TriangleMesh`` into ``Tensor``.
+
+Once again, in your terminal, enter:
+
+.. code-block:: python
+
+    >>> monkey = o3d.io.read_triangle_mesh(monkey_model.path)
```

Comment:
I would suggest changing to use `monkey_data` for the dataset and `monkey_mesh` for the triangle mesh or `monkey_model` for the triangle model. Same for the rest of the file and other data.

Reply:
Great idea !!!!

Done.